mutation_partition.cc

```diff
@@ -675,45 +675,45 @@ void write_counter_cell(RowWriter& w, const query::partition_slice& slice, ::ato
   });
 }
 
-    template<typename Hasher>
-    void appending_hash<row>::operator()(Hasher& h, const row& cells, const schema& s, column_kind kind, const query::column_id_vector& columns, max_timestamp& max_ts) const {
```

Comment:
With github review, you can temporarily change the diff view to ignore whitespace changes, which allows for easy verification of reformatting changes (not that there's a huge risk of error creeping into them).

Reply:
There is a settings icon on the right side, after "Jump to" with a checkbox for that

Reply:
Yes, that's what I was using.

Reply:
When viewing a PR in VSCode those indentation changes are also ignored. I guess this means we don't have to bother with these additional "restore formatting" patches any more.

Reply:
Mixing formatting with real changes is still back for some work (like backporting).